assets/scripts/Actor/Actor.ts

```diff
@@ -19,7 +23,7 @@ export default abstract class Actor extends cc.Component {
   @property(cc.Float)
   public maxSpeed = 0;
 
-  protected rigidBody: cc.RigidBody;
+  public rigidBody: cc.RigidBody;
```

Comment:
```suggestion
  protected rigidBody: cc.RigidBody;
```
se tiver que acessar métodos dessa propriedade de forma externa, crie funções públicas internas à classe, pq provavelmente vai ter reutilização